File: src/java/每日一题/二叉树的镜像.java
package 每日一题;

import MyTools.工具类.TreeNode;

public class 二叉树的镜像 {
    public void Mirror(TreeNode root) {
        if (root == null) {
            return;
        }
        TreeNode t = root.left;
        root.left = root.right;
        root.right = t;
        Mirror(root.left);
        Mirror(root.right);
    }
}

File: src/java/力扣题目/剑指offer/二叉树的镜像.kt
package 力扣题目.剑指offer

class 二叉树的镜像 {
    fun mirrorTree(root: TreeNode?): TreeNode? {
        root ?: return null
        val temp = root.left
        root.left = root.right
        root.right = temp
        mirrorTree(root.left)
        mirrorTree(root.right)
        return root
    }
}